### ISSUE TITLE
Fail to update device list

### ISSUE THREAD

**Issue by param0037** [LuxCoreRender/BlendLuxCore#871]
I am using BlendLuxcore 2.7 beta on Blender 3.6. I also selected CUDA on the addon properties. However, the renderer didn't work on either CPU or GPU. Even when I clicked update devices list, Luxcore reported an error"OpenCL driver API error (code: -1, file:D:\a\1\Luxcore\src\luxrays\devices\ocldevice.cpp, line: 89): CL_DEVICE_NOT_FOUND". If I switched to CPU, the same error occurred. My OS is Windows 10, CPU is Intel i7-9th, GPU is Nvidia GTX1660Ti.

**Comment by odil24** [comment]
Hi. Download v2.7beta1:
https://github.com/LuxCoreRender/BlendLuxCore/releases/tag/blendluxcore_v2.7beta1

**Comment by param0037** [comment]
Thanks but this is the version I tested. I tried reinstalling it again but the problems still exist. Here are the pictures of the errors, which I used CPU and clicked on "update device list" respectively.
![viewport](https://github.com/LuxCoreRender/BlendLuxCore/assets/93871506/0955b9b0-fb82-4dbb-ac28-6637e404fb2b)
![error_reported](https://github.com/LuxCoreRender/BlendLuxCore/assets/93871506/bb954c53-9ebf-4c33-ba1a-9582be0d55b2)

**Comment by odil24** [comment]
Reinstall again and update your GPU driver and try to install again

**Comment by param0037** [comment]
I reinstalled and updated the driver, but it didn't work, still the same error

**Comment by odil24** [comment]
It works perfect with my lower specs than yours in 3.6.7 also

**Comment by param0037** [comment]
Is Opencl environment necessary? I haven't installed that.

**Comment by odil24** [comment]
Not needed. Just need to update drivers, reinstall fresh build as I said previously and thats it

**Comment by odil24** [comment]
Not needed. Just need to update drivers, reinstall fresh build as I said previously and thats it

**Comment by odil24** [comment]
@neo2068 close an issue

**Comment by jkerman2000** [comment]
> Reinstall again and update your GPU driver and try to install again

I'm having the same problems, is it because I'm using the game-ready drivers and not the studio drivers, or is it something else?

**Comment by param0037** [comment]
Not really. I have updated my driver to studio but it still reports CL_DEVICE_NOT_FOUND, even when I use CPU.获取Outlook for Android
